### ISSUE TITLE
Figure out survey strategy to find out more about our users.

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.mozilla.org#1648]
As we continue to work on Thimble, we should make efforts to get to know our users better. We can gather feedback in a few different ways and should discuss what a good approach might be. Particularly I'd like to focus on Teachers.

**Goals**
To find out...

* What is working well
* What is causing problems
* What content are they currently using
  * Where are they finding it
  * Are they making it on their own?
* What content they would like to use

**Some options**

* Create a form with something like Google Forms or Surveymonkey
  * Tweet it out?
  * Include it in one of the Newsletters.
  * Throw a link to it on the Homepage
  * Include it at the end of support email exchanges.
* Add a feedback tab to the site

I'll draft up a question set.

**Comment by gideonthomas** [comment]
Clarification: is our focus group teachers in our network who currently use Thimble (clubs, hive, etc.) or teachers outside our network who currently use Thimble (networks like Ladies Learning Code, etc.)?

**Comment by flukeout** [comment]
I think whoever we can get access to - I don't think there'd be a big difference in their feedback and use cases. Since we're starting to talk to Gigabit folks, and they're interested in connecting us to teachers, I'll design a survey for that use case, and theme it around helping us figure out the vision.

**Comment by flukeout** [comment]
I think it's happening here..

https://github.com/flukeout/css-diner/blob/develop/js/restaurant.js#L550

``wrapperEl.append("&lt;" + elName +  "/&gt;");``

should be...

``wrapperEl.append("&lt;/" + elName +  "&gt;");

**Comment by humphd** [comment]
@flukeout wrong bug?

**Comment by flukeout** [comment]
@humphd Yeah, totally wrong - deleting all of our comments related to that now.
